chapters/unitexpressions.tex

```diff
@@ -39,15 +43,13 @@ \section{The Syntax of Unit Expressions}\doublelabel{the-syntax-of-unit-expressi
    "."
 \end{lstlisting}
 
-The ISO standard allows that a multiplication operator symbol is left
-out. However, Modelica enforces the ISO recommendation that each
-multiplication operator is explicitly written out in formal
-specifications. For example, Modelica does not support "\lstinline!Nm!" for
-newtonmeter, but requires it to written as "\lstinline!N.m!".
+The SI standard specifies that a multiplication operator symbol is written as space 
+or as a dot. For example, Modelica does not support "\lstinline!Nm!" for
+newton-meter, 
+but requires it to written as "\lstinline!N.m!". 
```

Comment:
For units the proper solution seems to be to put the quotation marks inside the inline code, since you write `unit="N.m"`, but not use this for e.g. division since division is part of the unit-string - and not the entire unit-string

Reply:
The general quote-issue is created as #2364